pkg/plugins/resources/postgres/store.go

```diff
@@ -80,8 +81,9 @@ func (r *postgresResourceStore) Create(_ context.Context, resource model.Resourc
 	}
 
 	version := 0
-	statement := `INSERT INTO resources VALUES ($1, $2, $3, $4, $5, $6);`
-	_, err = r.db.Exec(statement, opts.Name, "", opts.Mesh, resource.GetType(), version, string(bytes)) // todo(jakubdyszkiewicz) solve db migration
+	statement := `INSERT INTO resources VALUES ($1, $2, $3, $4, $5, $6, $7, $8);`
+	now := time.Now()
```

Comment:
Why `creationTime/modificationTime` is set at DAO layer instead of Service layer ?